src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -103,33 +103,38 @@ struct ComputeNormalDotFluxes {
  * \brief Computes the penalty flux
  *
  * The penalty flux is given by:
+ *
  * \f{align}
  * F^*(\Psi) =& F(\Psi) = 0 \\
- * F^*(\Pi) =& F(\Pi)_{\mathrm{int}} + \\
- *         \frac{1}{2} p \left(U^-_{\mathrm{ext}} - U^-_{\mathrm{int}}\right) \\
- * F^*(\Phi_i) =& F(\Phi_i)_{\mathrm{int}} - \\
- *         \frac{1}{2} p \left(U^-_{\mathrm{ext}} - U^-_{\mathrm{int}}\right)
- * \f}
- * where \f$F^*\f$ is the normal dotted with the numerical flux and \f$F\f$ is
- * the normal dotted with the flux, which is computed in
- * ScalarWave::ComputeNormalDotFluxes.
+ * F^*(\Pi) =& F(\Pi)_{\mathrm{int}} +
```

Comment:
So since this is the interior flux we're going with `F^i(\Pi)_{\mathrm{int}) n_{i,\mathrm{int}}` or `(F^i(\Pi)n_i)_{\mathrm{int}}`.

I need to go through and update that for everything else in the code. I'll try to do that ASAP :)

Reply:
I am confused by this comment. Nowhere in this doxy snippet documenting penalty flux do we have a formula with `F^i n_i` which would need annotation as you've highlighted... Is this meant to be a note or a suggestion or .. ?

Reply:
Oh, so when you write F(\Pi)_int it must be `(F^i(\Pi) n_i)_int`, otherwise I don't know what F is. Since The LHS is a scalar (G) and the second term with the char fields is a scalar, so should the F term. Does that clarify things?

Reply:
I see, you want the textual description that reads "_... F is the normal dotted with flux (computed in ScalarWave::ComputeNormalDotFluxes), ..._" to be spelled out by explicitly replacing `F` with `F^i n_i` etc. I've made this change, please see the fixup commit!